.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+
+node_js:
+- 'node'
+
+os:
+- linux
+- osx
+
+before_install:
+- if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;

```

Comment:
What is this for?

Reply:
It's for `npm test --silent` command, because this command needs launch a vscode window to execute unit test.

In headless linux, if we want to run any UI related command, need enable xvfb for that.

Reply:
That makes sense - thanks for the explanation!